dockers/docker-orchagent/swssconfig.sh

```diff
@@ -38,7 +38,7 @@ fast_reboot
 
 HWSKU=`sonic-cfggen -m /etc/sonic/minigraph.xml -v minigraph_hwsku`
 
-SWSSCONFIG_ARGS="00-copp.config.json ipinip.json mirror.json "
+SWSSCONFIG_ARGS="00-copp.config.json ipinip.json mirror.json ports.json "
```

Comment:
In comments to the design @lguohan suggested to make port speed mandatory in the minigraph. If for some reason it is not specified, switch will remain with the value from the SAI profile.